examples/restaurant_demo/restaurant.py

```diff
@@ -98,80 +101,92 @@ def get_private_counts_per_hour(self, privacy_budget: float) -> dict:
 
         # Only count the hours without the minutes (12:00 and 12:30 both add 1 to the 12 count)
         visits = self._hour_visits.copy()
-        visits["Time entered"] = pd.to_datetime(visits["Time entered"]).apply(lambda x: x.strftime(r'%H')).astype(int)
-        
+        visits["Time entered"] = pd.to_datetime(visits["Time entered"]).apply(
+            lambda x: x.strftime(r'%H')).astype(int)
+
         # TODO: Question, should this be the privacy budget? The original code sets epsilon to be log(3)
         x = dp.CountInt(LN_3)
 
         for time in visits["Time entered"].unique():
-            private_hours_count[time] = x.result(list(visits[visits["Time entered"] == time]["Time entered"].astype(int)))
-        
+            private_hours_count[time] = x.result(
+                list(visits[visits["Time entered"] == time]["Time entered"].astype(int)))
+
         return private_hours_count
 
     def get_non_private_counts_per_day(self) -> dict:
         day_counts = dict()
 
         for day in self._day_visits["Day"].unique():
-            day_counts[day] = self._day_visits[self._day_visits["Day"] == day]["Day"].count()
-        
+            day_counts[day] = self._day_visits[self._day_visits["Day"]
+                                               == day]["Day"].count()
+
         return day_counts
 
     def get_private_counts_per_day(self, privacy_budget: float) -> dict:
         # Pre-process the data set: limit the number of days contributed by a visitor to COUNT_MAX_CONTRIBUTED_DAYS
-        day_visits = bound_visits_per_week(self._day_visits, COUNT_MAX_CONTRIBUTED_DAYS)
+        day_visits = bound_visits_per_week(
+            self._day_visits, COUNT_MAX_CONTRIBUTED_DAYS)
 
         day_counts = dict()
 
         x = dp.CountInt(LN_3)
 
         for day in day_visits["Day"].unique():
-            day_counts[day] = x.result(list(day_visits[day_visits["Day"] == day]["Day"]))
-        
+            day_counts[day] = x.result(
+                list(day_visits[day_visits["Day"] == day]["Day"]))
+
         return day_counts
 
     def get_non_private_sum_revenue(self) -> dict:
```

Comment:
Do you think we should have a docstring for this method?

Also, since PyDP is in early days, it may be best to start using triple quotes/a delimiter (""") instead of a hash (#) for all docstrings. This can potentially help with auto-generating (API) docs.

Reply:
Do you mean for all comments or just doc strings?